### ISSUE TITLE
Small model for augmentations search

### ISSUE THREAD

**Issue by lolokoko28** [stfwn/ats-privacy-replication#8]
That is about first model from section 4.4, page 5ft: "Ms is used for
privacy quantification. It is trained only with 10% of the
original training set for 50 epochs. This overhead is equivalent to the training with the entire set for 5 epochs, which is
very small. "

- [x] Create script for creating small, reproductible subset of CIFAR100 for train and evaluation of small model for policy search.  (with seed)
- [x] Train small model on this subset
